.github/workflows/publish.yaml

```diff
@@ -27,6 +27,15 @@ on:
   # https://github.blog/2022-02-10-using-reusable-workflows-github-actions
   workflow_call:
     inputs:
+      manual_pull_request:
```

Comment:
This feels like a double-negative flag. I think it ought to be `open_pull_request` that has default True

Reply:
Yeah, I agree. Will change.